packages/core/strapi/src/core/registries/components.ts

```diff
@@ -0,0 +1,53 @@
+import { has } from 'lodash/fp';
+import type { Common, Schema } from '@strapi/types';
+
+const componentsRegistry = () => {
+  const components: Record<string, Schema.Component> = {};
```

Comment:
Should those be strings or components UIDs? (I see that you're using component UIDs below, hence the question)

Reply:
I can